docs/Hosts/Spawn-Hosts.md

```diff
@@ -57,51 +57,30 @@ notification](../Project-Configuration/Notifications#spawn-host-expiration) for
-when you'd like your unexpirable host to be automatically turned on or off. A sleep schedule lets you choose recurring
-times of the week when you want your host to be on, and when it's okay for it to be turned off. For example, you could
-set a sleep schedule where your host is up and running during your work hours from 10 am to 6 pm between Monday and
-Friday and otherwise let it turn it off overnight and during weekends. Setting a sleep schedule ensures your host is on
-during your working hours, while also stopping the host when it's not being actively used. Powering down hosts when
-they're not being used is important to ensure that hosts are being efficiently utilized and to avoid over-spending on
-idle long-lived hosts.
+Unexpirable hosts have a feature (which is enabled by default) called a sleep schedule that allows you to control when
+you'd like your unexpirable host to be automatically turned on or off. A sleep schedule lets you choose recurring times
+of the week when you want your host to be on, and when it's okay for it to be turned off. For example, you could set a
+sleep schedule where your host is up and running during your work hours from 10 am to 6 pm between Monday and Friday and
+otherwise let it turn it off overnight and during weekends. Setting a sleep schedule ensures your host is on during your
+working hours, while also stopping the host when it's not being actively used. Powering down hosts when they're not
+being used is important to ensure that hosts are being efficiently utilized and to avoid over-spending on idle
+long-lived hosts.
 
 Note that stopping the host during off hours means **shutting down the host**, not hibernating the host. If needed, see
 [the FAQ](#faq) for more info.
 
-**While this feature is being rolled out, please [choose and complete one of the three options](#beta-testing) for your
-unexpirable host(s). You must complete one of the options before the beta test ends at 11 am (Eastern Time) on Tuesday
-September 3. If you do not take any action, a default schedule will be set on your behalf for your unexpirable host(s)
-at the end of the beta test period.**
-
-### ***Important Note***: This Feature is in Beta Testing! {#beta-testing}
-
-**This feature is currently available for beta testing until September 3.** While it's in beta testing, _using
-the sleep schedule is an opt-in feature_. During this beta testing period, you have the following options to choose from
-for your host, **please pick and complete one before the beta test period ends**:
+**The beta test period for this feature has ended as of 11 am (Eastern Time) September 3, so all users who did not set a
+sleep schedule or request a permanent exemption have been assigned a default schedule. If you'd like your unexpirable
+host to use something other than the default settings, your options are:**
 
 1. [Option 1](#sleep-schedule) (Recommended for most users): Set a sleep schedule for each of your existing
```

Comment:
nit: [Option 1] and [Option 2] are not needed since the list is numbered.

Reply:
Done.